draft-pda-protocol.md

```diff
@@ -559,36 +559,39 @@ struct {
 * `payload` is the encrypted input share.
 
 To generate the report, the client begins by encoding its measurements as an
-input for the PA protocol and splitting it into a leader share `leader_share`
-and a helper share `helper_share`. (This procedure is specified by the PA
-protocol.) Each input share has the following format:
+input for the PA protocol and splitting it into input shares according to the
+protocol. Input shares have the following structure:
 
 ~~~
 struct {
-  select (PAParam.proto) { // PAParam for the PA task
-    case prio: PrioInputShare;
-    case hits: hitsInputShare;
+  select (PAParam.proto, aggregator_type) {
+    case prio, leader: PrioLeaderInputShare;
+    case prio, helper: PrioHelperInputShare;
+    case hits, leader: HitsLeaderInputShare;
+    case hits, helper: HitsHelperInputShare;
   }
 } PAInputShare;
 ~~~
 
-Next, the client generates an HPKE {{!I-D.irtf-cfrg-hpke}} context for the
-leader and helper by running
+[TODO: Not sure if the use of `aggregator_type` above is syntactically correct.]
```

Comment:
Yeah, you can't do this in the TLS syntax. I would just say this stuff verbally
"Of the appropriate share type for the protocol and role" down on line 593 below.

Reply:
Removed the PAInputShare structure altogether and clarified above that the the format of the input share depends on the protocol and aggreagtor role.